index.js

```diff
@@ -18,12 +18,30 @@ class ElectronPreferences extends EventEmitter2 {
 		super();
 
 		_.defaultsDeep(options, {
+			config: {
+				debounce: 150,
+			},
 			sections: [],
 			webPreferences: {
 				devTools: false,
 			},
 		});
 
+		this.options = options;
+
+		// Legacy: Set config values
+		if (options.css && !options.config.css) {
+
+			this.options.config.css = options.css;
+
+		}
+
+		if (options.dataStore && !options.config.dataStore) {
+
+			this.options.config.dataStore = options.dataStore;
```

Comment:
Same thing here, I'd add a warning message:
```suggestion
			console.warn("DEPRECATED: dataStore option has been deprecated and will be removed in a future version. It now lives under config.dataStore.");
			this.options.config.dataStore = options.dataStore;
```